.github/workflows/ci.yml

```diff
@@ -548,13 +491,6 @@ jobs:
         env:
           PYTHON_VERSIONS: ${{ needs.build-info.outputs.python-versions-list-as-string }}
           DEBUG_RESOURCES: ${{needs.build-info.outputs.debug-resources}}
-      - name: "Tests Pytest collection"
```

Comment:
This wil save ~ 4 minutes of "feedback time" as this test-pytest-collection is also repeated in each test job.